docs/source/getstarted.rst

```diff
@@ -88,13 +92,19 @@ Then, run the program.
 
 The code for the Hello World program can be `viewed here. <https://github.com/anki/cozmo-python-sdk/tree/master/examples/tutorials/01_basics/01_hello_world.py>`_
 
-We can edit this code to make Cozmo say something new. Let's write our first program using this code.
+.. important:: The usbmuxd (USB Multiplexor Daemon) must be installed on any computer system paired with an iOS device before running any programs. Installing `iTunes <http://www.apple.com/itunes/download/>`_ will also install the usbmuxd on those systems. Linux users can install the usbmuxd module through the command line with `sudo apt-get install usbmuxd`.
+
+.. important:: Make sure adb (Android Debug Bridge) is installed on your system prior to running a program with an Android device. See the :ref:`Install page <initial>` for instructions.
+
+You are now all set up to run Python programs on Cozmo. Next we will go over how to edit the above code to make Cozmo say something new. Let's write our first program.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 Next Steps - "Night-Night"
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Copy ``01_hello_world.py`` to a new file named ``nightnight.py``.
+.. important:: Do NOT save example programs to the *cozmo_sdk_examples* directory. Doing so will overwrite the original baseline programs.
```

Comment:
What's funny here is that we give them this warning, but then in step one we don't tell them how to avoid saving example programs to the cozmo_sdk_examples, directory, right?